builder.go

```diff
@@ -73,6 +88,11 @@ func (p MailBuilder) ToAddrs(to []mail.Address) MailBuilder {
 	return p
 }
 
+// GetTo returns the stored to addresses.
+func (p *MailBuilder) GetTo() []mail.Address {
+	return p.to
```

Comment:
Let's use the following style to return a copy of the to slice:

var dst []string
dst = append(dst, src...)